lib/shopify_api/resources/fulfillment_request.rb

```diff
@@ -0,0 +1,11 @@
+module ShopifyAPI
+  class FulfillmentRequest < Base
+    init_prefix :order
+
+    def order_id
+      @prefix_options[:order_id]
+    end
+
+    def cancel; load_attributes_from_response(post(:cancel, {}, only_id)); end
```

Comment:
Oh... I see. this is for cancelling a fulfillment *request*, not a fulfillment. Perhaps we should be more explicit about that as well, seeing as we'll actually be adding the ability to cancel the fulfillment itself. So `cancel_fulfillment_request`?